### PR TITLE
enable and disable paypal based on currency

### PR DESCRIPTION
**Pull request by ScottDowne**
WIP, needs some design updates to the credit card button.

### CODE REVIEW

**Comment by ScottDowne** [comment]
Done, has some css and tests.

@alicoding r?

File: components/payment-options.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+var PayPalButton = React.createClass({
+  mixins: [require('react-intl').IntlMixin],
+  propTypes: {
+    onClick: React.PropTypes.func,
+    submitting: React.PropTypes.bool,
+    validate: React.PropTypes.array,
+    submit: React.PropTypes.array.isRequired
+  },
+  onChange: function() {
+    this.props.onClick = this.props.onClick || function() {};
```
Comment (alicoding) [inline components/payment-options.jsx]: Will it ever be empty here?? Why not enforce the isRequired for this prop?

Comment (ScottDowne) [inline components/payment-options.jsx]: Yeah, currently in single-form:

```
+          <PayPalButton
+            submitting={this.state.submitting}
+            submit={["frequency", "amount"]}
+            validate={["amount"]}
+            onSubmit={this.paypal}
+          />
```

Comment (ScottDowne) [inline components/payment-options.jsx]: onClick is used in sequential form to toggle the credit card details on and off.

Comment (alicoding) [inline components/payment-options.jsx]: so sometimes you don't want to pass it in because it shouldn't do anything?

File: data/locale-data.js [hunk 2]
```diff
@@ -17,6 +18,7 @@ module.exports = {
     'en-US': 'US',
     'en-CA': 'CA',
     'en-GB': 'GB',
-    'it': 'IT'
+    'it': 'IT',
+    'in': 'IN'
```
Comment (alicoding) [inline data/locale-data.js]: This is not a locale.

File: tests/selenium/sequential-stripe-test.js [hunk 3]
```diff
@@ -8,11 +8,18 @@ module.exports = function(driver, By, done) {
     }
     driver.findElement(By.css('.page-active .next-button')).click();
     driver.wait(function() {
-      return driver.findElement(By.id('payment-cc-label')).isEnabled().then(function(enabled) {
+      if (url !== 'http://localhost:3000/in/') {
```
Comment (alicoding) [inline tests/selenium/sequential-stripe-test.js]: fix this as well.

File: components/payment-options.jsx (hunk 1 above)
Comment (ScottDowne) [inline components/payment-options.jsx]: Correct, we don't need it for certain cases.

**Comment by ScottDowne** [comment]
@alicoding Fixed up.

**Comment by alicoding** [comment]
looking good.

**Pull request merged**